design/CreateExperiment.md

```diff
@@ -5,29 +5,73 @@ tuned.
+  
 
-`performance_profile` consists of two parts:
+ `performance_profile` consists of two parts:
 
-* A yaml file defines the metrics to be monitored and the queries used to derive those metrics.
-* Source code that defines the performance objective using the metrics provided
+  - A yaml file defines the metrics to be monitored and the queries used to derive those metrics.
+  - Source code that defines the performance objective using the metrics provided
 
-Currently we have the following pre-defined profiles
+Currently, we have the following pre-defined profiles
 
 | Profile Name                           | yaml                                        | Associated Source | Comments                                       |
 |----------------------------------------|---------------------------------------------|-------------------|------------------------------------------------|
 | resource-optimization-openshift        | [resource-optimization-openshift yaml](https://github.com/kruize/autotune/blob/master/manifests/autotune/performance-profiles/resource_optimization_openshift.yaml)        | [ResourceOptimizationOpenshiftImpl.java](https://github.com/kruize/autotune/blob/master/src/main/java/com/autotune/analyzer/performanceProfiles/PerformanceProfileInterface/ResourceOptimizationOpenshiftImpl.java)            | This is used for Remote Monitoring Usecase     |
 | resource-optimization-local-monitoring | [resource-optimization-local-monitoring yaml](https://github.com/kruize/autotune/blob/master/manifests/autotune/performance-profiles/resource_optimization_local_monitoring.yaml) | [ResourceOptimizationOpenshiftImpl.java](https://github.com/kruize/autotune/blob/master/src/main/java/com/autotune/analyzer/performanceProfiles/PerformanceProfileInterface/ResourceOptimizationOpenshiftImpl.java)            | This is used for Local Monitoring Usecase     |
 | default                                | NA (User defined SLO)                       | [source](https://github.com/kruize/autotune/blob/master/src/main/java/com/autotune/analyzer/performanceProfiles/PerformanceProfileInterface/DefaultImpl.java)            | This is applicable to all to Autotune Usecases |
 
+***Note: One of Performance Profile name or SLO data needs to be present. Both cannot be present together!***
+
+### Experiment Type
+- **mode** \
+  It can be either `monitor` or `experiment` 
+- **target_cluster** \
+   Target cluster can be one of `remote` or `local`
+
+### Experiment Target
+- **kubernetes_objects** \
+   contains details of the deployment, containers and namespace of the application to be monitored. 
+  - **type** \
+    A type can either be a `Deployment`, `Replicaset`, `Statefulset`, `Deployment_Config`, `Replication_Controller`, 
+    `Daemonset`, etc.
+  - **name** \
+    A string name is specified for identifying individual deployments. Each deployment can have multiple experiments.
```

Comment:
Not sure what you mean by `Each deployment can have multiple experiments`? We currently support only one experiment. Also this should not be called `deployment`